src/locales/en.json

```diff
@@ -754,6 +754,8 @@
   "mno_enterprise.templates.dashboard.marketplace.show.launch_app": "Launch app",
   "mno_enterprise.templates.dashboard.marketplace.show.connect_app": "Connect App",
   "mno_enterprise.templates.dashboard.marketplace.show.price": "Price",
+  "mno_enterprise.templates.dashboard.marketplace.show.price.pricing_plan.type.one_off": "One-time payment",
+  "mno_enterprise.templates.dashboard.marketplace.show.price.pricing_plan.type.usage_based": "Usage based",
```

Comment:
If you add new locales in the dashboard, you need to update them in mnoe as well.
We already have this line:
```
"mno_enterprise.templates.dashboard.marketplace.show.pricing.payg": "Usage Based",
```
Is this used differently?